app/src/main/java/it/niedermann/owncloud/notes/persistence/NoteServerSyncHelper.java

```diff
@@ -395,9 +393,6 @@ protected void onPostExecute(LoginStatus status) {
             for (ICallback callback : callbacks) {
                 callback.onFinish();
             }
-            Log.d(TAG, "onPostExecute: Update widgets");
-            dbHelper.updateSingleNoteWidgets();
-            dbHelper.updateNoteListWidgets();
```

Comment:
I think this shouldn't be removed. When you receive changes from the server (new note, deleted notes or updated notes), the widgets should still be updated.